### DISCUSSION TITLE
Possibly make individual vault settings documentation clearer?

### DISCUSSION THREAD

**Discussion by handdara** [nvim-telekasten/telekasten.nvim#323] [Ideas]
Hi, I was wondering why my tags weren't seeming to work while in any vault besides the main. Took me a second but I guessed after seeing that the vault also had the "home" setting that each vault has to be configured just as the main settings to use the same tags. And it seems like that was correct. No problems there, I was just thinking that maybe the readme or help doc could make this a little more obvious? Maybe it's just me though and to most people the readme section is more obvious to others. I personally assumed that the main settings (in the top level of the `require('telekasten').setup()` arg table) would trickle down. 

Specifically talking about [here](https://github.com/renerocksai/telekasten.nvim?tab=readme-ov-file#vaults) maybe something along the lines of a list of the settings that can be set for the main and sub-vaults separately and then that top level settings don't trickle down.

Just a thought, as it may save some others from the same confusion, but I could've also just missed something or just be weirdly wired. Still loving the plugin, getting miles and miles of use out of it so thanks!

**Comment by lambtho12** [comment]
I do not personally use the vault functionality and did not implement it myself, so I cannot really help for that. 

So I will trust you on that. If you feel that things need to be added to the doc, please propose a PR with changes in the `doc/telekasten.txt` and `README.md` files to improve it.
